webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9718,6 +9717,27 @@ var userOverride = {
                 }
                 return '';
             };
+            _this.getMessageCreatedAtTime = function (createdAtTime) {
+                if (TIME_FORMAT_24_HOURS) {
+                    var msgtime = new Date(createdAtTime);
+                    var yesterday = new Date();
```

Comment:
why is it named yesterday?

Reply:
When I was trying the code- I had to check if the message is of yesterday or not- so I gave the name yesterday that time - 
should I change this?

Reply:
give a name that convey's what is it for

Reply:
done